src/components/MappingEvents/Statistics.js

```diff
@@ -30,9 +30,10 @@ const Statistics = ({
   // translator: Description for number of days left in the mapping event
   const daysLeftLabel = t`Verbleibende Tage`;
```

Comment:
Unrelated: did we switch to german at some point?

Reply:
No. Probably my oversight. Do we always go english by default?

Reply:
If we change this now, until this will get translated in transifex, it will show english text though. It that acceptable for a period of time? Our mapping event users currently are only in Germany?

Reply:
I will change this to english by default